erpnext/public/js/utils.js

```diff
@@ -1,6 +1,6 @@
 // Copyright (c) 2015, Frappe Technologies Pvt. Ltd. and Contributors
 // License: GNU General Public License v3. See license.txt
-frappe.provide("erpnext");
+frappe.provide("erpnext ");
```

Comment:
```suggestion
frappe.provide("erpnext");
```
why is this changed?